### DISCUSSION TITLE
Feb 12

### DISCUSSION THREAD

**Discussion by ngsy** [GymHubCommunity/GymHub-FE#147] [DailyMeeting]
### ⚽️ 회의 내용

- 개발 현황
수 : 운동기록 페이지 작업 했음.
수영 : 로컬 서버 띄우기 및 axios 작업 
윤혁 : 설정페이지 , 메인페이지 디자인 작업 진행 중 
슬희 :로그인 페이지 디자인 , 기능 , 오류 해결, 회원가입 기능 작업중 
- 프론트 진행상황 디자이너 공유 
 -전체 회의때 보여주는걸로
- 픽시드 플러팅 버튼 눌렀을때 어떤식으로 나올지 화면이필요함,
- 스웨거 배포 해주셨으면 좋겠다. 인텔리제이는 PTSD 온다.

### 🏀 오늘 할 일

수 : 로컬서버 연동, 운동기록 마무리하고 , 팔로우 기능 
수영 : axios 오늘 올리고, 운동지도 디자인 작업 
윤혁 : 메인페이지 디자인 작업 , 
슬희 :회원가입 디자인 , 기능 할 예정 

### 🎱 TMI

수 : 친구가 왓차, 라프텔, 웨이브 아이디를 빌려줘서 볼게 너무 많아서 아이패드 거치대를 구매후 침대에서 애니를 보고있습니다. 겹치는게 많아서 다른  사람은 돈주고 사지않았으면 좋겠다.환승연애 재미없다. 내 취향 아님 난 환승 안함.
수영 : 망했습니다. 연휴 동안 운동을 못갔습니다. 헬스장 이제 가려구요 
윤혁 : 오늘 어께랑 하체 하실 예정 , 5일 연속으로 술을 마셨다. 금주를 하겠다. 아니다 줄이겠다. 오.. 술크업 
슬희 :아침에 어떤 사람에게 카톡이 왔다. 해외에서 카톡이 왔다. 링크드인을 통해 번호를 알고 카톡으로 연락이왔다.

**Comment by hanseulhee** [comment]
짐허브에 인증해주세용
